components/suggest/src/suggestion.rs

```diff
@@ -175,6 +166,20 @@ impl Suggestion {
             _ => None,
         }
     }
+
+    pub fn score(&self) -> f64 {
+        match self {
+            Self::Amp { score, .. }
+            | Self::Pocket { score, .. }
+            | Self::Amo { score, .. }
+            | Self::Yelp { score, .. }
+            | Self::Mdn { score, .. }
+            | Self::Weather { score, .. }
+            | Self::Fakespot { score, .. }
+            | Self::Exposure { score, .. } => *score,
+            Self::Wikipedia { .. } => DEFAULT_SUGGESTION_SCORE,
```

Comment:
I really like how this doesn't have a default branch.  It makes it a lot more difficult to forget to update it when we add a new suggestion.